src/main/java/com/example/auctionapp/model/Item.java

```diff
@@ -0,0 +1,59 @@
+package com.example.auctionapp.model;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "items")
+public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @Column(name = "user_id")
+    private Long userId;
+
+    @NotNull
+    @Column
+    private String name;
+
+    @Column(name = "category_id")
+    private Long categoryId;
+
+    @Column(name = "subcategory_id")
+    private Long subcategoryId;
+
+    @Column
+    private String description;
+
+    @Column
+    private byte[] photo;
```

Comment:
Most probably, images will be stored somewhere else, other than our DB, so this can be text, as we will only store img url